11LessonEleven/homework/css/inspired.css

```diff
@@ -0,0 +1,75 @@
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    /*font-size: 100%;
+    font: inherit;*/
+    vertical-align: baseline;
+}
+
+header{
+  background-image:url("../img/bg.jpg");
+  height:700px;
+  background-repeat:no-repeat;
+  background-size:cover;
+  display:flex;
+  flex-direction:column;
+  justify-content:center;
+}
+
+header h1, header p{
+  color:white;
+  padding-left:5%;
+}
+
+header p{
+  padding-right:50%;
+}
+
+header h1{
+  padding-bottom:20px;
+  padding-right:50%;
+}
+section{
+  background-color:lightgrey;
+  width:100%;
+  display:flex;
+
+}
+
+article h3, article p{
+  margin-left:10%;
+  margin-right:10%;
+}
+
+article h3{
+  padding-top:40px;
+  padding-bottom:20px;
+  font-size:1.5em;
+}
+
+article p{
+  padding-bottom:40px;
+}
+
+@media screen and (max-width: 640px) {
```

Comment:
Remember, with the mobile-first mindset we start small and work our way up; that means we use `min-width` instead of max.
min-width: 320, 480, 620, 810, 1440.